src/lib/site-import/status.js

```diff
@@ -275,16 +275,14 @@ ${ maybeExitPrompt }
 
 						importJob.progress = { status: jobStatus, steps: jobSteps };
 					}
-				} else {
-					if ( ! importJob ) {
-						return resolve( 'No import job found' );
-					}
-
-					( {
-						progress: { status: jobStatus, steps: jobSteps },
-					} = importJob );
+				} else if ( ! importJob ) {
+					return resolve( 'No import job found' );
 				}
 
+				( {
+					progress: { status: jobStatus, steps: jobSteps },
+				} = importJob );
+
 				createdAt = importJob.createdAt;
```

Comment:
Maybe we could use the same syntax as above to be consistent?

```js
( {
    createdAt,
    completedAt,
    progress: { status: jobStatus, steps: jobSteps },
} = importJob );
```
Or maybe the same syntax as below?

```js
jobStatus = importJob.progress.jobStatus;
jobSteps = importJob.progress.jobSteps;
createdAt = importJob.createdAt;
completedAt = importJob.completedAt;
```

Reply:
Good idea, the deconstruction can be somewhat confusing in that format anyway. Changed in 7fc429e